### PR TITLE
refactor(blade): rebrand carousel

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: packages/blade/src/components/Carousel/NavigationButton/StyledNavigationButton.native.tsx [hunk 1]
```diff
@@ -28,6 +31,7 @@ const StyledNavigationButton = ({
       onPress={onClick}
       onPressIn={() => setIsPressed(true)}
       onPressOut={() => setIsPressed(false)}
+      style={castNativeType(theme.elevation.lowRaised)}
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Carousel/NavigationButton/StyledNavigationButton.native.tsx:34]: we added shadow to nav buttons now? it wasn't there before. is this intended?

File: packages/blade/src/components/Carousel/types.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-type BackgroundColorString<T extends ColorObjects> = `${T}.background.${DotNotationColorStringToken<
-  Theme['colors'][T]['background']
->}`;
-type BrandColorString = `brand.${DotNotationColorStringToken<Theme['colors']['brand']>}`;
-type OverlayColor = BackgroundColorString<'surface'> | BrandColorString;
+type InteractiveTokens = DotNotationToken<Theme['colors']['interactive']['background']>;
+type FeedbackTokens = DotNotationToken<Theme['colors']['feedback']['background']>;
+type SurfaceTokens = DotNotationToken<Theme['colors']['surface']['background']>;
+type OverlayTokens = DotNotationToken<Theme['colors']['overlay']>;
+type OverlayColor =
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Carousel/types.ts:10]: why so many options for overlay color?

File: packages/blade/src/components/Carousel/NavigationButton/StyledNavigationButton.native.tsx (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/Carousel/NavigationButton/StyledNavigationButton.native.tsx:34]: I think it was missed before.

File: packages/blade/src/components/Carousel/types.ts (hunk 2 above)
Comment (anuraghazra) [reply packages/blade/src/components/Carousel/types.ts:10]: Yes previously we also exposed all the feedback, surface etc colors.

File: packages/blade/src/components/Carousel/types.ts [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import type { Theme } from '~components/BladeProvider';
 import type { BoxProps } from '~components/Box';
 import type { Platform } from '~utils';
-import type { DotNotationColorStringToken } from '~utils/types';
+import type { DotNotationToken } from '~utils/lodashButBetter/get';
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Carousel/types.ts:4]: Why is `DotNotationToken` in `lodashButButter/get`?

**Comment by chaitanyadeorukhkar** [comment]
Why are the quotes on carousel in yellow? They were in blue before.

Before:
Screenshot 2023-12-27 at 3 38 23 PM

Now:
Screenshot 2023-12-27 at 3 38 08 PM

File: packages/blade/src/components/Carousel/types.ts (hunk 3 above)
Comment (anuraghazra) [reply packages/blade/src/components/Carousel/types.ts:4]: It's a new improved type that I wrote, we will refactor out the older DotNotationColorStringToken things and replace it with this one later.

**Comment by anuraghazra** [comment]
Changed the tokens for it in storybook to `theme.colors.feedback.text.information.subtle`

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**